definitions/v3/blutopia-api.yml

```diff
@@ -54,10 +54,24 @@ search:
   paths:
     # https://hdinnovations.github.io/UNIT3D-Community-Edition-Docs/api_endpoints.html
     # https://github.com/HDInnovations/UNIT3D-Community-Edition/blob/master/app/Http/Controllers/API/TorrentController.php
-    - path: "/api/torrents/filter?api_token={{ .Config.apikey }}&name={{ if .Query.IMDBID }}{{ else }}{{ .Keywords }}{{ end }}{{ if .Query.TMDBID }}&tmdbId={{ .Query.TMDBID }}{{ else }}{{ end }}{{ if .Query.IMDBIDShort }}&imdbId={{ .Query.IMDBIDShort }}{{ else }}{{ end }}{{ if  .Query.TVDBID }}&tvdbId={{ .Query.TVDBID }}{{ else }}{{ end }}&sortField={{ .Config.sort }}&sortDirection={{ .Config.type }}&perPage=100&page=1{{ range .Categories }}&categories[]={{.}}{{end}}{{ if .Config.freeleech }}&free=1{{ else }}{{ end }}"
+    - path: "/api/torrents/filter?"
       response:
         type: json
         attribute: attributes
+  inputs:
+  # if we have an id based search, add Season and Episode as query in name for UNIT3D < v6.  Else pass S/E Params for UNIT3D >= v6
+    $raw: "{{ range .Categories }}categories[]={{.}}&{{ end }}{{ if .Config.freeleech }}freeleech=1&{{ else }}{{ end }}"
+    sortField: "{{ .Config.sort }}"
+    sortDirection: "{{ .Config.type }}"
+    perPage: 100
+    page: 1
+    api_token: "{{ .Config.apikey }}"
+    name: "{{ if or (.Query.TVDBID) (.Query.IMDBID) (.Query.TMDBID) (.Query.Episode) }}{{ else }}{{ .Keywords }}{{ end }}"
```

Comment:
if `.Episode` is available we still want to pass the series title term or keyword or else we get random results

Reply:
Really if keywords were passed in the search, even if an id was also passed, why wouldn't we use then in any scenario? It could be a user trying to limit results further. if the Indexer supports it we should use what comes in

Reply:
Fair point